index.js

```diff
@@ -267,7 +267,11 @@ Filter.prototype.isDirectory = function(relativePath, entry) {
   var srcDir = this.inputPaths[0];
   var path = srcDir + '/' + relativePath;
 
-  return (entry || fs.lstatSync(path)).isDirectory();
+  try {
+    return (entry || fs.lstatSync(path)).isDirectory();
+  } catch (e) {
+    return false;
```

Comment:
I don’t believe that generally swallowing all possible errors is the right path forward here. We need to better understand the circumstances that are causing the failure first.